### ISSUE TITLE
Changes in ArduinoJson broke Sender.cpp

### ISSUE THREAD

**Issue by CaioFPeres** [universam1/iSpindel#607]
## I'm submitting a ...
  - [ X ] bug report

## Report

ArduinoJson, a requirement library, changed their syntax from major version 5 to 6, as can be seen in these pages:

Version 5: https://arduinojson.org/v5/faq/how-to-know-the-type-of-a-value/
Version 6: https://arduinojson.org/v6/doc/upgrade/#aschar-and-ischar-1

Since the project has a library requirement of ArduinoJson version higher than 5, it is currently bugging to this change of syntax, in case the user gets the latest ArduinoJson version.

In short, the file affected is Sender.cpp, at line 501:

`if (kv.value().is<char*>())`

Changing to:

`if (kv.value().is<const char*>())`

Fixes the issue.
